src/e2e/java/teammates/e2e/cases/e2e/StudentProfilePageE2ETest.java

```diff
@@ -33,9 +26,9 @@ public void testAll() {
 
         ______TS("Typical case: Log in with filled profile values");
```

Comment:
Should we keep this style of describing test cases? On a side note, do you think it is necessary to add a description to `testAll()` somewhere since we are removing descriptions for all `testAll()` functions?